includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -185,6 +191,12 @@ private function get_module_block_html( $block ) {
 			<section class="wp-block-sensei-lms-course-outline-module">
 				<header class="wp-block-sensei-lms-course-outline-module__name">
 					<h2 class="wp-block-sensei-lms-course-outline__clean-heading">' . $block['title'] . '</h2>
+					<div
```

Comment:
This is probably going to be heavily refactored when we connect to the actual state so I did a very simple soltion for now.